web/ui/two-steps-page.svelte

```diff
@@ -5,6 +5,26 @@
 
   let prevTitle = document.title
 
+  let stepFirstElement: HTMLDivElement
+  let stepSecondElement: HTMLDivElement
+
+  function scrollToTop(element: HTMLDivElement) {
+    if (element instanceof HTMLElement) {
+      element.scrollTo({
+        top: 0,
+        behavior: 'smooth'
+      })
+    }
+  }
+
+  export function scrollFirstToTop() {
+    scrollToTop(stepFirstElement)
```

Comment:
Seems like if we will replace `scrollToTop` to `stepFirstElement?.scrollTo` we will make code simpler. The helper for the single function is not very useful and only makes system more complicated.